addons/core/defines.hpp

```diff
@@ -13,6 +13,7 @@
 #define RADIO_OWNER 7
 #define SPEAKER_OFFSET 8
 #define POWER_OFFSET 9
+#define HALFDUPLEX_OVERRIDE_OFFSET 10
```

Comment:
This is a serious API change. Some other people store Data at the End of that array. Even though they shouldn't.
Need to specify clearly that the first 30 elements of that Array are reserved for TFAR use.
#TODO for me.